livekit-android-test/src/test/java/io/livekit/android/room/participant/LocalParticipantMockE2ETest.kt

```diff
@@ -591,20 +591,21 @@ class LocalParticipantMockE2ETest : MockE2ETest() {
     }
 
     @Test
-    fun publishWithNoResponseCausesException() = runTest {
+    fun publishWithNoResponseReturnFalseWithoutException() = runTest {
```

Comment:
This will affect the setMicrophone/Camera/Screenshare functions, these should return a boolean if we're removing the exception to signal that the set failed.